src/views/SalesInfo/index.tsx

```diff
@@ -0,0 +1,101 @@
+];
+
+const SalesInfo: React.FC = () => {
+  return (
+    <>
+      <Spacer size={'md'} />
+      <Card>
+        <InnerCardContainer borderBottom={false}>
+          <StyledContentContainer>
+            <StyledContentTextContainer>
+              <H1 color={colors.white}>wPOKT Liquidity bootstrapping Event</H1>
+            </StyledContentTextContainer>
+            <StyledChartContainer>
+              <BarChart width={600} height={300} data={data} margin={{ top: 20, right: 30, left: 20, bottom: 20 }}>
+                <XAxis xAxisId={0} dataKey="name" hide />
+                <XAxis xAxisId={1} dataKey="name" tickMargin={10} />
+                <YAxis domain={[0, 50000]} />
+                <Tooltip cursor={{ fill: 'transparent' }} />
+                <Bar barSize={10} xAxisId={0} dataKey="max" fill={colors.green} />
+                <Bar barSize={10} xAxisId={1} dataKey="min" fill={colors.white}>
+                  <ErrorBar dataKey="error" width={4} strokeWidth={2} stroke={colors.green} />
+                  <ErrorBar dataKey="errorNegative" width={4} strokeWidth={2} stroke={colors.red} />
+                </Bar>
+              </BarChart>
+            </StyledChartContainer>
+            <StyledContentTextContainer>
+              <P1 color={colors.white}>Price movement if nobody buys</P1>
+              <Spacer size={'xs'} />
+              <P2 color={colors.red}>89% wPokt sold</P2>
+            </StyledContentTextContainer>
+          </StyledContentContainer>
+          <Spacer size={'md'} />
+          <StyledInfoCard>
+            <P2 color={colors.white} center={true}>
+              {' '}
+              Welcome to the wPOKT distribution event. This distribution is the first opportunity to acquire wPOKT that
+              will enable holders to participate in wPOKT. The distribution will occur on Balancer using a Liquidity
+              Bootstrapping Pool. Before participating in wPOKT, we highly encourage users to read all of the wPOKT
+              documentation and information located in the links below. Welcome aboard and happy farming.
+            </P2>
+            <Spacer size={'md'} />
+            <H2 color={colors.white} center={true}>
+              WPOKT: 0,00006373 USDC/ DAI
+            </H2>
+          </StyledInfoCard>
+          <Spacer size={'md'} />
+          <StyledContentContainer>
+            <StyledContentTextContainer>
+              <H1 color={colors.white}>Resources</H1>
+            </StyledContentTextContainer>
+            <StyledButtonContainer>
+              <div>
+                <BuyWpoktSvg />
+              </div>
+              <div>
+                <BuyWpoktSvg />
+              </div>
+              <div>
+                <BuyWpoktSvg />
+              </div>
+              <div>
+                <BuyWpoktSvg />
+              </div>
+              <div>
+                <BuyWpoktSvg />
+              </div>
+            </StyledButtonContainer>
```

Comment:
Are these supposed to be buttons?

Reply:
Yup, my mistake! Fixed in 2fce3762b6ccecc37e40ff55f2aa1c25e9bbdf03